pkg/router/f5/plugin_test.go

```diff
@@ -1415,7 +1410,7 @@ func TestF5RouterPartition(t *testing.T) {
 		if !ok {
 			t.Fatalf("Test case %q missing partition key %s", tc.name, name)
 		}
-		defer mockF5.close()
+		mockF5.close()
```

Comment:
Hmm, you are right - defer is delayed till the function returns, so would need a closure here in order to do that. Yeah, its safer to keep what you have there.